src/NuGet.Clients/NuGet.VisualStudio.Implementation/Utility/FrameworkNameUtility.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Versioning;
+using NuGet.Frameworks;
+
+namespace NuGet.VisualStudio.Implementation.Utility
+{
+    public static class FrameworkNameUtility
```

Comment:
After inspecting the code, the existing pattern is to use `new FrameworkName(NuGetFramework.DotNetFrameworkName)` and `NuGetFramework.Parse(FrameworkName.ToString())`. These are simple solutions so I will remain consistent and do it this way (and remove my utility thing).